cmd/atlas/atlas.go

```diff
@@ -55,7 +55,7 @@ To learn more, see our documentation: https://www.mongodb.com/docs/atlas/cli/sta
 // loadConfig reads in config file and ENV variables if set.
 func loadConfig() error {
 	if err := config.LoadAtlasCLIConfig(); err != nil {
-		return fmt.Errorf("error loading config: %w", err)
+		return fmt.Errorf("error loading config: %w. Please run `atlas config init` to reconfigure your profile", err)
```

Comment:
[nit] I'm not sure we want to recommend config init, since if customers don't fix their file, they won't be able to do anything:

```shell
~/workplace/mongocli CLOUDP-201369 *29 ?6 ❯ atlas config init                                                          1.21.3 19:41:30
2023/10/26 19:41:34 error loading config: While parsing config: toml: expected character =
```

maybe we want to say  "Please make sure you have a valid yaml config file at "pathToFile"" or something along those lines?

Reply:
I tested locally and `config init` recreates the file, removing the old config fully.  A user who knows how to fix the issue can use `config edit`, but this gives users with little knowledge the ability to get going again.

Goal of the message is to help them get to steady state, rather than having to figure out what to do next.

Reply:
Just added an example in the description to show how it all works.

Reply:
ah, nevermind me, I must have run config init in the wrong version!